src/store/actions.ts

```diff
@@ -154,7 +154,7 @@ export default {
     commit('updateVariables', variableMap)
   }),
   loadGridVariables: tryAction(async ({ state, commit, getters } : { state: ApplicationState, commit: any, getters: Getters}) => {
-    state.isGridLoading = true
+    commit('setIsGridLoading', true)
     commit('updateGridVariables', [])
```

Comment:
Can the grid variables ever be [] without a call loading?

Reply:
UpdateGridVariables is set to an empty array so that during loading old data wont be visible.